### ISSUE TITLE
Crash opening a log

### ISSUE THREAD

**Issue by rynowak** [KirillOsenkov/MSBuildStructuredLog#230]
Hit a crash while opening a log (see attached)

```
---------------------------

---------------------------
Error while analyzing build. Sorry about that. Please Ctrl+C to copy this text and file an issue on https://github.com/KirillOsenkov/MSBuildStructuredLog/issues/new 

System.NullReferenceException: Object reference not set to an instance of an object.

   at Microsoft.Build.Logging.StructuredLogger.CompilationWrites.TryParse(Task task)

   at Microsoft.Build.Logging.StructuredLogger.ManagedCompilerTask.get_CompilationWrites()

   at Microsoft.Build.Logging.StructuredLogger.DoubleWritesAnalyzer.AnalyzeTask(Task task)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeTask(Task task)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Analyze()

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeBuild(Build build)
---------------------------
OK   
---------------------------
```
[MonoSanity_Debug_AnyCPU_Build_2018-12-29T13_46_33.1198788-08_00.zip](https://github.com/KirillOsenkov/MSBuildStructuredLog/files/2716742/MonoSanity_Debug_AnyCPU_Build_2018-12-29T13_46_33.1198788-08_00.zip)

**Comment by KirillOsenkov** [comment]
This should be fixed in 2.0.64. Start the viewer from Start Menu by searching for "MSB" and wait for it to auto-update:
![image](https://user-images.githubusercontent.com/679326/50542493-55caa800-0b73-11e9-9c8f-8577e3a7dc12.png)

**Comment by rynowak** [comment]
awesome! thanks Kirill!
